libs/payments/stripe/src/lib/stripe.util.ts

```diff
@@ -95,3 +96,31 @@ export const getSubscribedProductIds = (subscribedPlans: StripePlan[]) => {
     ? subscribedPlans.map(({ product: productId }) => productId as string)
     : [];
 };
+
+/**
+ * Formats a Stripe Invoice to the FirstInvoicePreview DTO format.
+ */
+export function stripeInvoiceToFirstInvoicePreviewDTO(
```

Comment:
Instead of adding this function to `stripe.utils.ts` could it instead be added into it's own file? I.e. `lib/util/stripeInvoiceToFirstInvoicePreviewDTO.ts`. I believe this is a new pattern we are trying to follow.